charts/keycloak/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: keycloak
-version: 10.1.0
+version: 10.1.1
```

Please bump the minor version for a new feature

```suggestion
version: 10.2.0
```